cart.html

```diff
@@ -0,0 +1,716 @@
+                            fill="#A6A6A6" />
+                          <path d="M9.45532 7.33333V11.3333" stroke="#A6A6A6" stroke-width="2" stroke-linecap="round"
+                            stroke-linejoin="round" />
+                          <path d="M13.0046 7.33333V11.3333" stroke="#A6A6A6" stroke-width="2" stroke-linecap="round"
+                            stroke-linejoin="round" />
+                        </svg> Eliminar
+                      </button>
+                      <div class="product__add-card-counter">
+                        <button type="button"
+                          class="product__add-card-counter-button product__add-card-counter-button--less-add js-substract-unit-to-product" data-action="substract-to-cart">
+                          <svg xmlns="http://www.w3.org/2000/svg" width="26" height="30" viewBox="0 0 26 30" fill="none">
+                            <path d="M5.9574 15H20.4539" stroke="#D8D8D8" stroke-width="2" stroke-linecap="round"
+                              stroke-linejoin="round" />
+                          </svg>
+                        </button>
+                        <input class="product__add-card-number js-product-add-card-number" type="number" name="quantity"
+                          value="1"></input>
+                        <button type="button"
+                          class="product__add-card-counter-button product__add-card-counter-button--more-add js-add-unit-to-product" data-action="add-to-cart">
+                          <svg xmlns="http://www.w3.org/2000/svg" width="26" height="30" viewBox="0 0 26 30" fill="none">
+                            <path d="M12.7163 6.6V23.4" stroke="#D8D8D8" stroke-width="2" stroke-linecap="round"
+                              stroke-linejoin="round" />
+                            <path d="M5.46802 15H19.9645" stroke="#D8D8D8" stroke-width="2" stroke-linecap="round"
+                              stroke-linejoin="round" />
+                          </svg>
+                        </button>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </ul>
+          </div>
+          <div class="cart__information-purchase-summary">
+            <h4 class="cart__information-purchase-summary-title">Resumen de compra</h1>
```

Comment:
- Abres un `h4`, pero cierras con `h1`
- No hay un `h3` antes, por lo cual esta rompiendose la jerarquia
```suggestion
            <h3 class="cart__information-purchase-summary-title">Resumen de compra</h3>
```